tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -11,8 +11,10 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         curr_time = repr(time.time()).replace('.', '')
 
-        self.contact = MockContact(givenName='Name%s'%curr_time[12:], name='Name%s'%curr_time[12:], familyName='')
+        for contact in self.data_layer.sim_contacts:
```

Comment:
I think this deletion needs to be discussed. At the moment we don't delete anything on the SIM card when we run the tests (except a targeted delete in the tearDown of this test). I know some people like @mermi store their contacts in the SIM cards while running the tests.

@mwargers Do you think it's possible to make the test more stable without this deletion?